src/common/lz4_compression.h

```diff
@@ -0,0 +1,24 @@
+// Copyright 2019 yuzu Emulator Project
+// Licensed under GPLv2 or any later version
+// Refer to the license.txt file included.
+
+#include <vector>
+
+#include "common/common_types.h"
+
+namespace Common::Compression {
+
+// Compresses a source memory region with LZ4 and returns the compressed data in a vector. If
+// use_LZ4_high_compression is true, the LZ4 subalgortihmn LZ4HC is used with the highest possible
```

Comment:
This is no longer relevant, also:

```suggestion
// use_LZ4_high_compression is true, the LZ4 subalgortihm LZ4HC is used with the highest possible
```

These functions should likely also have doxygen compatible descriptions.